tests/drivers/flash_api/src/main.c

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2021-2024 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/ztest.h>
+#include <zephyr/drivers/flash.h>
+#include <zephyr/device.h>
+
+/** Test instrumentation **/
+/*
+ * Below structure gathers variables that may be used by simulate API calls to mock behaviour
+ * of a flash device; the variables may be used however it is desired, because the it is
+ * up to mocked function and check afterwards to relate a variable to result
+ */
```

Comment:
```suggestion
 * The structure below gathers variables that may be used by the simulated API calls to mock behavior
 * of a flash device; the variables may be used however it is desired, because it is
 * up to the mocked functions and checks afterwards to relate these variables to results
 */
```
maybe better?